src/components/RequestBox/RequestBox.jsx

```diff
@@ -6,8 +6,8 @@ import buildNewRxRequest from '../../util/buildScript.2017071.js';
 import MuiAlert from '@mui/material/Alert';
 import Snackbar from '@mui/material/Snackbar';
 import { shortNameMap, ORDER_SIGN, PATIENT_VIEW } from '../../util/data.js';
-import { getAge, createMedicationDispenseFromMedicationRequest, createMedicationFromMedicationRequest } from '../../util/fhir.js';
-import { retrieveLaunchContext, prepPrefetch } from '../../util/util.js';
+import { getAge, createMedicationDispenseFromMedicationRequest, createMedicationFromMedicationRequest, getDrugCodeableConceptFromMedicationRequest } from '../../util/fhir.js';
+import { retrieveLaunchContext, prepPrefetch, getMedicationSpecificCdsHooksUrl } from '../../util/util.js';
```

Comment:
```suggestion
import { retrieveLaunchContext, prepPrefetch, getMedicationSpecificEtasuUrl } from '../../util/util.js';
```
since request-generator complains 
```
RequestBox.jsx:340  Uncaught (in promise) ReferenceError: getMedicationSpecificEtasuUrl is not defined
```

also on line 228 (the `if` condition), I have a code suggestion:
```js
if (response.data.parameter?.[0]?.resource.contained) {
```

otherwise request-generator complains
```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading '0')
```

Reply:
I have fixed these issues, thanks for finding the issue with submitting the prescription to pims.